xtask/src/ast_src.rs

```diff
@@ -394,6 +652,25 @@ pub(crate) const AST_SRC: AstSrc = AstSrc {
             T![;]
         }
 
+        // TODO: clarify whether this does include assoc type with bounds
```

Comment:
yes

Reply:
Thanks, removed it in 80c529d1dc57873ad07ed1eb656eafe72c9a7e29